lib/elastic_apm/spies/action_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module ElasticAPM
+  # @api private
+  module Spies
+    # @api private
```

Comment:
I don't think you need this for modules/classes that're only there for the namespacing.

Reply:
Did you find out a way to get Rubocop into RubyMine btw?

Reply:
Yeah, I have Rubocop in RubyMine but don't always have it enabled because it's distracting.
We have the comments `api private` for all the other namespace declarations and rubocop complains if I don't have the comments for `api private`. So I think we have to leave them.